src/yival/experiment/app/app.py

```diff
@@ -191,8 +191,6 @@ def generate_combo_metrics_data(
 
             sample_count = 0
             for group in group_experiment_results:
-                if sample_count >= 3:
```

Comment:
have you test headline_generation and animal_story with sample > 3 ?

Reply:
Yes
it works well